src/test/denialofservice_tests.cpp

```diff
@@ -190,6 +190,68 @@ BOOST_AUTO_TEST_CASE(stale_tip_peer_management)
     connman->ClearTestNodes();
 }
 
+BOOST_AUTO_TEST_CASE(block_relay_only_eviction)
+{
+    const CChainParams& chainparams = Params();
+    auto connman = std::make_unique<ConnmanTestMsg>(0x1337, 0x1337, *m_node.addrman);
+    auto peerLogic = PeerManager::make(chainparams, *connman, *m_node.addrman, nullptr,
```

Comment:
This has been used before too, but can we name it `peerman` instead, like `connman`? Sounds more clear.

Reply:
Yes, will do if I push again, though this should be probably done everywhere in this test per scripted-diff.